cfy_manager/components/cli/cli.py

```diff
@@ -33,6 +34,17 @@
 PROFILE_NAME = 'manager-local'
 
 
+@contextmanager
+def _hide_logs():
+    """Increase the logging level, so that the password isn't in the logfile"""
+    current_level = get_file_handlers_level()
+    set_file_handlers_level(logging.ERROR)
```

Comment:
Not something to change in this PR, but... this is really not good, we need to do something better about this.

Reply:
(and I know it's just copied from elsewhere here, just noting it as it rather draws attention)

Reply:
hm, change how?